pkgs/development/python-modules/websockets_client/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, buildPythonPackage, fetchPypi,
+  six, isPy27 }:
+buildPythonPackage rec {
+  pname = "websocket_client";
+  version = "0.47.0";
+
+  src = fetchPypi {
+    inherit version pname;
+    sha256 = "0jb1446053ryp5p25wsr1hjfdzwfm04a6f3pzpcb63bfz96xqlx4";
+  };
+
+  prePatch = if isPy27 then ''
```

Comment:
I think `if` is not needed here, it doesn't harm to remove it for all Python versions

Reply:
yes, it is not necessary. It is more a way to tell maintainers it was only expected for python-2.7, but I can happily drop it if you prefer.

Reply:
Well, since I don't think Python 2 support will be droppedin the near future, it is more confusing than helpful.

Reply:
I have dropped it